lightwood/helpers/torch.py

```diff
@@ -27,6 +27,25 @@ class LightwoodAutocast:
     """
     Equivalent to torch.cuda.amp.autocast, but checks device compute capability
     to activate the feature only when the GPU has tensor cores to leverage AMP.
+
+    **Attributes:**
+
+    * `active` (bool): Whether AMP is currently active.
+
+    **Methods:**
+
+    * `__init__(self, enabled=True)`: Initializes the class and sets the initial value of `active`.
```

Comment:
Please add these docstrings inside each of the methods they describe.

Reply:
Once method descriptions have been moved, please remove them from this docstring.